canonical-web-and-design/maas-ui
# Display deploy form fields horizontally.

## Done

- Display deploy form fields horizontally.
- Update form files to TypeScript.

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)

### QA steps

- Select some deployable machines in the machine list and use the take action menu to open the deploy form.
- Check the fields are horizontal and look like the design: https://app.zeplin.io/project/5ee104736f620ab91f8c5c97/screen/5ef0f486199b1b460886853d

## Fixes

Fixes: https://github.com/canonical-web-and-design/maas-squad/issues/2031.
